### PR TITLE
fixes dryRun bug: updates settleappt in UG

### LINKED ISSUE #151: [Bug] Wrong suggested command given
Recommended 'missedappt' command instead of 'missappt'.

<hr><sub>[original: Cronyxx/ped#6]<br/>
</sub>

### LINKED ISSUE #159: Settle invalid command
Did you mean settleappt?

<hr><sub>[original: sreesubbash/ped#7]<br/>
</sub>

### PR DESCRIPTION
**Pull request by woon17**
close #159
close #151

### CODE REVIEW

File: src/main/java/seedu/address/commons/core/Messages.java [hunk 1]
```diff
@@ -13,7 +13,7 @@
     public static final String MESSAGE_PERSON_NOT_FOUND = "The person '%1$s' is not registered";
     public static final String MESSAGE_INVALID_ROOM = "the room does not exist";
     public static final String MESSAGE_INVAILD_REFERENCE_ID =
-            "The reference ID '%1$s' does not belong to any registered person!";
+            "The referenceId '%1$s' does not belong to any registered person!";
```
Comment (SakuraBlossom) [inline src/main/java/seedu/address/commons/core/Messages.java]: As mentioned in pull request #165. For command results, "referenceId" should be 2 words. So "reference Id"

File: src/main/java/seedu/address/commons/core/Messages.java [hunk 2]
```diff
@@ -23,9 +23,12 @@
     public static final String MESSAGE_INVALID_INDEX = "The appointment index provided is not correct";
     public static final String MESSAGE_NOT_PATIENTLIST = "appointment(s) displayed do not belong to the same patient\n";
     public static final String MESSAGE_NOT_STAFFLIST =
-            "duty shift(s) displayed do not belong to the same staff doctor\n";
-    public static final String MESSAGE_NOT_MISSEDLIST = "current displaying appointment are not "
-            + "belong to missed appointment list\n" + "please type command: missedappt";
-    public static final String MESSAGE_NOTHING_SETTLE = "no missed appointment needs to settle";
+            "Duty shift(s) displayed do not belong to the same staff doctor\n";
```
Comment (SakuraBlossom) [inline src/main/java/seedu/address/commons/core/Messages.java]: As mentioned in pull request #165. 
First. Please check your grammar.
Second, rephrase to "Safety check: Duty shift(s) displayed must belong to the same staff doctor \n"

File: src/main/java/seedu/address/commons/core/Messages.java [hunk 3]
```diff
@@ -23,9 +23,12 @@
...
+    public static final String MESSAGE_NOT_MISSEDLIST =
+            "Only missed appointments can be settled.\n"
+                    + "Plsease first display the missed appointment listing, "
```
Comment (SakuraBlossom) [inline src/main/java/seedu/address/commons/core/Messages.java]: As mentioned in pull request #165. 
"Plsease" check your spelling

File: src/main/java/seedu/address/logic/parser/appointments/SettleAppCommandParser.java [hunk 4]
```diff
@@ -52,7 +52,7 @@ public ReversibleActionPairCommand parse(String args) throws ParseException {
         try {
             if (lastShownList.size() == 0) {
                 throw new ParseException(Messages.MESSAGE_NOTHING_SETTLE + "\n"
-                        + "No need: " + "settleappt " + args);
+                        + "No need: " + "settleappt" + args);
```
Comment (SakuraBlossom) [inline src/main/java/seedu/address/logic/parser/appointments/SettleAppCommandParser.java]: Change this to just "throw new ParseException(MESSAGE_NOTHING_SETTLE);"

File: src/main/java/seedu/address/commons/core/Messages.java [hunk 5]
```diff
@@ -23,9 +23,12 @@
...
+                    + "using missappt, before using settleappt command.";
```
Comment (SakuraBlossom) [inline src/main/java/seedu/address/commons/core/Messages.java]: Please put `missappt` in brackets.

File: src/main/java/seedu/address/commons/core/Messages.java [hunk 6]
```diff
@@ -23,9 +23,12 @@
...
+
+    public static final String MESSAGE_NOTHING_SETTLE = "No missed appointment needs to settle";
```
Comment (SakuraBlossom) [inline src/main/java/seedu/address/commons/core/Messages.java]: Rephrase and move this to your SettleAppCommandParser.java

File: src/main/java/seedu/address/commons/core/Messages.java (hunk 1 above)
Comment (woon17) [reply src/main/java/seedu/address/commons/core/Messages.java]: okay

File: src/main/java/seedu/address/commons/core/Messages.java (hunk 2 above)
Comment (woon17) [reply src/main/java/seedu/address/commons/core/Messages.java]: for this pr, I only edit settleappt command, duty shift relate command, I will edit in other pr

File: src/main/java/seedu/address/commons/core/Messages.java (hunk 3 above)
Comment (woon17) [reply src/main/java/seedu/address/commons/core/Messages.java]: okay.

File: src/main/java/seedu/address/logic/parser/appointments/SettleAppCommandParser.java (hunk 4 above)
Comment (woon17) [reply src/main/java/seedu/address/logic/parser/appointments/SettleAppCommandParser.java]: solved in other pr

File: src/main/java/seedu/address/commons/core/Messages.java (hunk 5 above)
Comment (woon17) [reply src/main/java/seedu/address/commons/core/Messages.java]: solved

File: src/main/java/seedu/address/commons/core/Messages.java (hunk 6 above)
Comment (woon17) [reply src/main/java/seedu/address/commons/core/Messages.java]: solved

**Pull request closed without merging**